domain/src/main/java/org/oppia/domain/OnboardingFlowController.kt

```diff
@@ -0,0 +1,60 @@
+package org.oppia.domain
+
+import androidx.lifecycle.LiveData
+import org.oppia.app.model.OnboardingFlow
+import org.oppia.data.persistence.PersistentCacheStore
+import org.oppia.util.data.AsyncResult
+import org.oppia.util.data.DataProviders
+import org.oppia.util.logging.Logger
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Controller for persisting and retrieving the user onboarding information of the app. */
+@Singleton
+class OnboardingFlowController @Inject constructor(
+  cacheStoreFactory: PersistentCacheStore.Factory, private val dataProviders: DataProviders,
+  private val logger: Logger
+) {
+  private val appHistoryStore = cacheStoreFactory.create("onboarding_flow", OnboardingFlow.getDefaultInstance())
```

Comment:
Rename is variable to `onboardingFlowStore`

Reply:
updated to onBoardingFlowStore